packages/gafl-webapp-service/package.json

```diff
@@ -47,7 +47,7 @@
     "debug": "^4.1.1",
     "find": "^0.3.0",
     "govuk-frontend": "^3.6.0",
-    "moment": "^2.25.1",
+    "moment": "^2.25.3",
```

Comment:
To update libraries you can use the lerna update tool from the root of the repo to keep versioning consistent in all projects.
`lernaupdate --non-interactive --dependency "moment@latest"` will update all projects that use it and rewrite package.json/package-lock.json files as necessary